alot/buffers.py

```diff
@@ -261,8 +261,13 @@ def rebuild(self, reverse=False):
             order = self.sort_order
 
         try:
+            exclude_tags = settings.get_notmuch_setting('search', 'exclude_tags')
+            if exclude_tags:
+                exclude_tags = [t for t in exclude_tags.split(';') if t]
+
             self.pipe, self.proc = self.dbman.get_threads(self.querystring,
-                                                          order)
+                                                          order,
+                                                          exclude_tags)
         except NotmuchError:
```

Comment:
Which command are we expecting to raise a `NotmuchError`? Does `settings.get_notmuch_setting` allow them to propagate up? If not then the `exclude_tags` bit out of the try/except.